src/main/resources/google_checks.xml

```diff
@@ -13,7 +13,7 @@
     To completely disable a check, just comment it out or delete it from the file.
     To suppress certain violations please review suppression filters.
 
-    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
+    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov, Mauryan Kansara ( Zops ).
```

Comment:
Please keep only real name

Reply:
removed. Added my name next to Max's, maintaining a specific order of: `contributor's names` first and then `mentor's names`.